test_examples/ephemeral-spark/README.md

```diff
@@ -0,0 +1,32 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Is this necessary for this use case or just leftovers from make lint?

Reply:
Apparently it's not strictly necessary, but I guess it doesn't hurt to have it.

Reply:
That being said, it'd be good to have more explanation on what this folder is doing as well